UserInterface/MenuItemDemos/MenuItemDemos/MenuItemDemos/MenuItemDemos.csproj

```diff
@@ -0,0 +1,43 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <ProduceReferenceAssembly>true</ProduceReferenceAssembly>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DebugType>pdbonly</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Xamarin.Forms" Version="4.1.0.555618" />
+  </ItemGroup>
+
+  <ItemGroup>
```

Comment:
Again, just wondering why VS gives these two XAML pages embedded resource entries, but not others.

Reply:
Actually, my guess is it's the templates. The project template provides App.xaml/MainPage.xaml and doesn't include embedded resource entries for them in the .csproj. Whereas the ContentPage item template does add embedded resource entries for new XAML files. That's my guess, anyway.

Reply:
That's probably what it is. To add pages (both Xaml and Code) I CTRL + SHIFT + A (or right-click and choose Add > New Item) and then choose Content Page or Content Page (C#). This must inject the entries in the csproj.

I'd suggest no change here since this is default VS behavior?